account/delete-account.md

```diff
@@ -10,6 +10,10 @@ keywords:
 - delete
 ---
 
-In the Clever Cloud Web Console, select **Profile** in the bottom left menu, then select **Delete my account** in the top menu of the freshly opened page.
+To delete your account:
+1. Open the Clever Cloud Web Console.
+2. Select **Profile** in the bottom left menu.
+3. Select **Delete my account** in the top menu of the freshly opened page.
+4. Click the red **Send me a confirmation email**. 
```

Comment:
It seems like there is a trailing space here, can you please drop it? You usually can configure your editor to display those